src/ir/cost.h

```diff
@@ -66,17 +66,30 @@ struct CostAnalyzer : public Visitor<CostAnalyzer, Index> {
     return ret;
   }
   Index visitLocalGet(LocalGet* curr) { return 0; }
-  Index visitLocalSet(LocalSet* curr) { return 1; }
+  Index visitLocalSet(LocalSet* curr) { return 1 + visit(curr->value); }
   Index visitGlobalGet(GlobalGet* curr) { return 1; }
-  Index visitGlobalSet(GlobalSet* curr) { return 2; }
+  Index visitGlobalSet(GlobalSet* curr) { return 2 + visit(curr->value); }
```

Comment:
Thanks for noticing these missing things. Good to fix them.

I think we may want to do a larger rewrite here eventually, to automatically visit children, but not before the boilerplate-removing PRs stuff lands.